setup/install-surelog.sh

```diff
@@ -6,10 +6,14 @@ src_path=$(cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
 sudo apt-get install -y build-essential cmake git pkg-config tclsh swig uuid-dev libgoogle-perftools-dev python3 python3-dev
 sudo apt-get install -y default-jre
 
-cd ${src_path}/..
-git submodule update --init --recursive third_party/tools/surelog
-cd third_party/tools/surelog
+mkdir -p deps
+cd deps
+
+git clone https://github.com/chipsalliance/Surelog.git
+cd Surelog
```

Comment:
Same comment regarding clearly auto documenting hash dependencies (for developers and end users).